src/test/java/io/neonbee/NeonBeeTest.java

```diff
@@ -120,7 +120,7 @@ void testDecorateEventbus() throws Exception {
         when(eventBus.addOutboundInterceptor(Mockito.any(Handler.class))).thenReturn(eventBus);
         ArgumentCaptor<Handler<DeliveryContext<Object>>> inboundHandlerCaptor = ArgumentCaptor.forClass(Handler.class);
         ArgumentCaptor<Handler<DeliveryContext<Object>>> outboundHandlerCaptor = ArgumentCaptor.forClass(Handler.class);
-        NeonBee.decorateEventBus(neonBee);
+        neonBee.decorateEventBus();
         verify(eventBus).addInboundInterceptor(inboundHandlerCaptor.capture());
         verify(eventBus).addOutboundInterceptor(outboundHandlerCaptor.capture());
         TrackingInterceptor inboundHandler = (TrackingInterceptor) inboundHandlerCaptor.getValue();
```

Comment:
Unrelated: for testFilterByProfile you can remove the type arguments. Taht would makes the test more compact and readable since each assertion fits into one line like:

```
    void testFilterByProfile() {
        assertThat(NeonBee.filterByAutoDeployAndProfiles(CoreVerticle.class, List.of(CORE))).isTrue();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(CoreVerticle.class, List.of(CORE, STABLE))).isTrue();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(CoreVerticle.class, List.of(STABLE))).isFalse();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(StableVerticle.class, List.of(STABLE))).isTrue();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(StableVerticle.class, List.of(STABLE, CORE))).isTrue();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(IncubatorVerticle.class, List.of(INCUBATOR))).isTrue();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(SystemVerticle.class, List.of(CORE))).isFalse();
        assertThat(NeonBee.filterByAutoDeployAndProfiles(SystemVerticle.class, List.of(ALL))).isFalse();
    }
```